test/e2e/tests/billing/testPaymentMethods.js

```diff
@@ -34,7 +34,11 @@ export default addTestNamePrefixes({
       .fillInput('@cardMonthInput', '11')
       .fillInput('@cardYearInput', '22')
       .fillInput('@cardCVCInput', '777')
-      .click('@addPaymentButton')
+      .click('@addPaymentButton');
+
+    client.pause(3000);
```

Comment:
please don't use pause, you should wait for some part of dom to be visible/hidden